editions/tw5.com/tiddlers/saving/Saving.tid

```diff
@@ -1,6 +1,6 @@
 created: 20140912140651119
 modified: 20220401151525812
-saving-browser: Firefox Chrome Edge [[Internet Explorer]] Safari Opera
+saving-browser: Firefox Chrome Edge [[Internet Explorer]] Safari Opera [[Standalone App]]
```

Comment:
I think, we should remove "Internet Explorer" from the list. Microsoft does the same: https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support